### ISSUE TITLE
Transformation Error - Stitch integration with Zendesk Chat

### ISSUE THREAD

**Issue by ruchit12** [singer-io/tap-zendesk-chat#39]
Hello, I have Zendesk chat integration to Snowflake Via Stitch, from past two days I am receiving an error and the data is not loaded. Can you please help me with it. The error I am receiving is: main - INFO Exit status is: Discovery succeeded. Tap failed with code 1 and error message: "Errors during transform". Target succeeded.
[Error.txt](https://github.com/singer-io/tap-zendesk-chat/files/8134389/Error.txt)

**Comment by vorpal56** [comment]
The fix is to change the webpath property in the catalog that's generated in discovery mode. This is likely due to some changes on Zendesk's part, though not entirely sure.

```
"webpath": {
    "type": ["null", "array"],
    "items": {
        "type": ["null", "object", "array"],
        "properties": {
            "timestamp": {
                "type": ["null", "string"],
                "format": "date-time"
            },
            "to": {
                "type": ["null", "string"]
            },
            "from": {
                "type": ["null", "string"]
            },
            "title": {
                "type": ["null", "string"]
            }
        },
        "items": {
            "type": ["null", "string", "integer"]
        },
        "additionalProperties": false
    }
}
```

**Comment by amyfeng-klaviyo** [comment]
We got the same error yesterday in stitch integration, any update on this?

**Comment by Vi6hal** [comment]
Hi guys,
This issue is fixed in the following P.R https://github.com/singer-io/tap-zendesk-chat/pull/41
and released in version 0.3.2
